src/consensus/validation.h

```diff
@@ -15,7 +15,7 @@
 /** A "reason" why something was invalid, suitable for determining whether the
   * provider of the object should be banned/ignored/disconnected/etc.
```

Comment:
In commit "[validation] Add CValidationState subclasses" (907ef02667b6c224df1457b67b7d7950cd58494f)

A previous version of this commit changed "something" to "a transaction" and "object" to "tx". Would be nice to keep these changes.

Reply:
fixed